test/mail_ssl_manual.bats

```diff
@@ -101,7 +101,7 @@ function teardown_file() {
 @test "checking ssl: manual cert changes are picked up by check-for-changes" {
     printf 'someThingsChangedHere' \
       >>"$(pwd)/test/test-files/ssl/${DOMAIN_SSL_MANUAL}/with_ca/ecdsa/key.ecdsa.pem"
-    sleep 10
+    sleep 15
```

Comment:
Is this change relevant to this PR or the logs?

Reply:
Rather this PR :)